sources/packages/forms/src/form-definitions/sfaa2022-23.json

```diff
@@ -5447,6 +5447,26 @@
           "properties": {},
           "lockKey": true,
           "defaultValue": "365"
+        },
+        {
+          "input": true,
+          "tableView": true,
+          "key": "currentTaxYear",
+          "label": "currentTaxYear",
```

Comment:
Please have the Current Tax Year label name as descriptive and not as currentTaxYear